### DISCUSSION TITLE
How can i do to let it answer the question

### DISCUSSION THREAD

**Discussion by thouyear** [oobabooga/text-generation-webui#3471] [Q&A]
![image](https://github.com/oobabooga/text-generation-webui/assets/138983583/02f99323-de07-41f3-a90f-5652e4585838)
![image](https://github.com/oobabooga/text-generation-webui/assets/138983583/2ea5247d-e361-4d67-8333-7c4476696084)
I have successfully loaded the model, but why can't the model reply ？

**Comment by missionfloyd** [comment]
According to [this](https://huggingface.co/ziqingyang/chinese-alpaca-2-7b/discussions/1), this is the prompt template for that model.
```
[INST] <<SYS>>
You are a helpful assistant. 你是一个乐于助人的助手。
<</SYS>>

INPUT_HERE [/INST]
```
And here's a quantized version, it might run better.
https://huggingface.co/keyfan/chinese-alpaca-7b-gptq

**Reply by thouyear** [reply]
Thank you very much for your answer.I have deployed the complete version of the model according to the tutorial.see https://github.com/ymcui/Chinese-LLaMA-Alpaca-2
![image](https://github.com/oobabooga/text-generation-webui/assets/138983583/2f4e373f-67fc-42b3-a01b-66519f5b34ee)
Now that the webui is running and prompts that the model has been loaded, theoretically, should this model work？
